doc/user/repositories.xml

```diff
@@ -94,15 +94,15 @@ int main() { printf("Hello, world!\n"); }
     Multiple calls to the &Repository; method
     will simply add repositories to the global list
     that &SCons; maintains,
-    with the exception that &SCons; will automatically eliminate
+    with the exception that &SCons; will automatically filter out
     the current directory and any non-existent
     directories from the list.
 
     </para>
 
   </section>
 
-  <section>
+  <section id="sec-repository-finding-sources">
```

Comment:
should this be `sect-`?

Reply:
That was the intent. Doesn't break anything this way, but I'll adjust.